### PR TITLE
Add dlp debug logs

### PR DESCRIPTION
**Pull request by rdenarie**

### CODE REVIEW

**Review by AzmiTouil — DISMISSED** [review]
LGTM

**Review by ahmadmido — DISMISSED** [review]
LGTM

File: core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java [hunk 1]
```diff
@@ -68,31 +70,54 @@
...
   }
-
+  
+  private boolean isInTrash(String entityId) {
+    ExtendedSession session = null;
+    try {
+      session = (ExtendedSession) WCMCoreUtils.getSystemSessionProvider().getSession(COLLABORATION_WS, repositoryService.getCurrentRepository());
+      Node node = session.getNodeByIdentifier(entityId);
+      boolean result = trashService.isInTrash(node);
+      LOGGER.debug("Item {} isEditorOpened={}", entityId, result);
```
Comment (ahamdi) [inline core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java]: I think you need to update the log message

Comment (rdenarie) [reply core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java]: doen, thanks @ahamdi

**Review by ahamdi — APPROVED** [review]

**Pull request merged**